tests/back/Acceptance/Product/InMemoryProductRepository.php

```diff
@@ -71,7 +71,7 @@ function (ProductInterface $product) use ($id) {
 
     public function findAll()
     {
-        throw new NotImplementedException(__METHOD__);
+        return array_values($this->products->toArray());
```

Comment:
```suggestion
        return $this->products->getValues();
```

But it's really the same, feel free to ignore this :wink:

Reply:
Sorry, I will ignore as my CI is green :laughing:.
Plus, this implementation is already done in the PIM 4.0.